tutorials/navigation/navigation_using_navigationmeshes.rst

```diff
@@ -0,0 +1,325 @@
+.. _doc_navigation_using_navigationmeshes:
+
+Using NavigationMeshes
+======================
+
+.. warning::
+
+    This chapter is a stub for linking and unfinished.
```

Comment:
```suggestion
    This chapter is a stub for linking and unfinished.
```

Thats a bit of a weird note IMO, what exactly is missing? Lots of docs pages could use fleshing out, and a lot of stuff is missing or outdated. IME, we don't leave similar notes everywhere. Open to hear opinions about this however, just not sure how useful it is.

Reply:
removed it